src/local_insight_engine/services/processing_hub/text_processor.py

```
@@ -146,36 +148,81 @@ def _process_chunk(self, chunk_data: Dict, document_id) -> TextChunk:
+
+        # Add neutralized text content for Q&A (but keep it factual and anonymized)
+        # This preserves key information while avoiding copyright issues
+        if original_text and len(original_text.strip()) > 50:
+            if bypass_anonymization:
+                # In factual mode, preserve original text for scientific content
+                neutralized_text = original_text
+            else:
+                # Simple neutralization: replace proper nouns with entity labels but keep structure
+                neutralized_text = self._create_neutral_summary(original_text, entities)
+            if neutralized_text:
+                neutralized_parts.append(f"\nNeutralized content summary:")
```

Ruff (0.12.2)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)